src/bin/drone.rs

```diff
@@ -60,6 +62,14 @@ fn main() {
                 .takes_value(true)
                 .help("request limit for time slice"),
         )
+        .arg(
+            Arg::with_name("wait")
+                .short("w")
+                .long("wait")
+                .value_name("NUMBER")
```

Comment:
clap is pretty awesome...  I recommend you pass something that looks like help to value_name().

when you ask for help with this code you see something like 
--wait NUMBER   : time to wait .....

What units?  Why does this thing need "leader info"? 

Were I the user, I'd prefer something that conveyed the units and better captured semantics, e.g.: 

--timeout SECS   wait at most SECS seconds to get necessary gossip from the network